src/main/java/org/desarrolladorslp/technovation/dto/MessagesReceiversDTO.java

```diff
@@ -0,0 +1,21 @@
+package org.desarrolladorslp.technovation.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Id;
+import java.util.UUID;
+
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class MessagesReceiversDTO {
```

Comment:
This class represents a single user ... please rename the class to `MessageReceiverDTO`